model/textparse/openmetricsparse_test.go

```diff
@@ -497,7 +525,7 @@ func TestOpenMetricsParseErrors(t *testing.T) {
 		},
 		{
 			input: `custom_metric 1 # {aa="bb"}`,
-			err:   "metric name custom_metric does not support exemplars",
+			err:   "expected value after exemplar labels, got \"EOF\"",
```

Comment:
This seems to duplicate the test above now.

Reply:
It's a little different since `custom_metric_total` belongs to a summary that was supported before while `custom_metric` was not. Please let me know and I remove it.

Reply:
I think it at least deserves a comment - there is a long series of `custom_metric_total` before and after this one `custom_metric`, so I would not expect future maintainers to recognize the specific need for this test.

Personally I would delete it.

Reply:
👍🏼 Deleted.